src/odemis/odemisd/modelgen.py

```diff
@@ -922,7 +1010,7 @@ def read_yaml(self, f):
             logging.warning("Settings file reset after loading failed with Exception %s." % ex)
             data = {}
         return data
-        
+
```

Comment:
Same error as in https://github.com/delmic/odemis/pull/1803#discussion_r654189045.
I need to fix this, but I have already updated the PR because this seems like a minor change.